rosbag2/src/rosbag2/writer.cpp

```diff
@@ -14,6 +14,9 @@
 
 #include "rosbag2/writer.hpp"
 
+#include <rosbag2_storage/filesystem_helper.hpp>
```

Comment:
Isn't that being flagged by cpplint?

Reply:
cpplint seems to think its a c-header and wants it before the rest of the headers.

Reply:
right. That's why we normally include the package headers with `"` instead of `<>`